src/App.tsx

```diff
@@ -1,47 +1,110 @@
-import React, { FC } from 'react';
+import React, { FC, useEffect, useState } from 'react';
 import './App.css';
-import Header from './components/Header';
+import axios from 'axios';
+import queryString from 'query-string';
+import { Innholdsformasjon } from './types';
 import CollapsedBox from './components/CollapsedBox';
-import { TitleLabel, AuthorLabel, ISBNLabel, ImageContainer } from './components/CustomElements';
-import pelsjegere from './resources/pelsjegere.jpg';
+import Header from './components/Header';
+import { AuthorLabel, ErrorTextField, ImageContainer, ISBNLabel, TitleLabel } from './components/CustomElements';
 
 const URL = window.location.href;
+const apiUrl = process.env.REACT_APP_INNHOLDSTJENESTE_API_URL;
+const imageUrl = process.env.REACT_APP_INNHOLDSTJENESTE_IMAGES_URL;
+const queryParams = queryString.parse(window.location.search);
 
 const App: FC = () => {
-  const shortSummary =
-    'Etter å ha brutt opp fra sin sakførerpraksis i Norge, reiste Ingstad til arktiske Canada, der han levde i fire år som pelsjeger.';
-  const longSummary =
-    'Etter å ha brutt opp fra sin sakførerpraksis i Norge, reiste Ingstad til arktiske Canada, der han levde i fire år som pelsjeger. Pelsjegerliv er beretningen om hans opplevelser i denne tiden. Boken ble utgitt i 1931  og er hans første av i alt tolv bøker';
-  const tableOfContents = [
-    'The opening',
-    'Second breakfast',
-    "Three's a crowd",
-    'May the fourth be with you',
-    'Cinco de mayo',
-  ];
+  const [innholdsinformasjon, setInnholdsinformasjon] = useState<Innholdsformasjon | undefined>();
+  const [errorPresent, setErrorPresent] = useState<boolean>(false);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+
+  useEffect(() => {
+    if (queryParams.isbn === '') {
+      setErrorPresent(true);
+      setErrorMessage(`Resource not found. \nParameter specifying isbn was not provided.`);
+      return;
+    }
+    const url = apiUrl + '?isbn=' + queryParams.isbn;
+    axios
+      .get(url)
+      .then(({ data }) => {
+        const innholdResponse = JSON.parse(data.body);
+        if (!innholdResponse) {
+          return;
+        }
+        innholdResponse.description_short = splitOnTag('<p>', innholdResponse.description_short).map(removeAllTags);
+        innholdResponse.description_long = splitOnTag('<p>', innholdResponse.description_long).map(removeAllTags);
+        innholdResponse.table_of_contents = splitOnTag('<br>', innholdResponse.table_of_contents).map(removeAllTags);
+        setInnholdsinformasjon(innholdResponse);
+        setErrorPresent(false);
+      })
+      .catch(() => {
+        setErrorPresent(true);
+        setErrorMessage('Failed to retrieve the resource, please try again.');
```

Comment:
Jeg ville vurdert å putte alle axios-response i en egen mappe for backend-kall. I dette tilfellet hatt axios kall som returner Promise<Innhold>.  Så klart har dere bare ett par kall så er ikke det så farlig. Men det øker gjenbrukbarheten av koden. 

Hvis du ønsket å putte dette kallet også i async await, men sleit fordi useEffecte begynte å klage, så kan jeg vise hvordan du kommer rundt det. Per og jeg benytter oss av async ... await for lesbarhetens skyld.